docs/Platform_Development.rst

```diff
@@ -1956,6 +1956,30 @@ The table **SOC\_STREAM\_ITEMS** stores informations of the items of the activot
 | UPDATED_DATE          | TIMESTAMP       | The last updated date of the stream item.                                          |
 +-----------------------+-----------------+------------------------------------------------------------------------------------+
 
+The table **SOC\_STREAM\_ITEMS** stores the items of the activity stream. Its purpose is to improve performances by precalculating the items that must be displayed in activity streams and to avoid to always recalculate them from the others tables.
+Each item references an activity, in the field **ACTIVITY\_ID**, with a specific type, stored in the field **STREAM\_TYPE**.
+When an activity is created, one or more items are created in the table **SOC\_STREAM\_ITEMS** (so an activity can be referenced by one or more items):
+- activity posted in the user stream: an entry is inserted in the table **SOC\_STREAM\_ITEMS** with the type **POSTER** and the owner id set to the user who has created the activity.
+- activity posted in a space: 2 entries are inserted in the table **SOC\_STREAM\_ITEMS**:
+  - an entry of type **POSTER** and with an **OWNER\_ID** equal to the space in which the post has been done
+  - an entry of type **POSTER** and with an **OWNER\_ID** equal to the user who posted the activity
+- comment on a space activity: no entry is added to table **SOC\_STREAM\_ITEMS** 
+  but the **UPDATED\_DATE** field is updated for the two entries described above.
+  
+For each activity created, an entry is added to the **SOC\_STREAM\_ITEMS** table and 
```

Comment:
The lines started from here must be removed, since all cases must be described in the above list (it must be completed, all cases are not covered yet).

Reply:
removed

Reply:
> (it must be completed, all cases are not covered yet)

This is not done yet. The list must be completed to describe all cases.

Reply:
@mdiha can you please help me to list the missing items: mentioning, liking....

Reply:
@mzorai the complete list of stream types by order is: SPACE, POSTER, LIKER, COMMENTER, MENTIONER, COMMENT_LIKER